### ISSUE TITLE
`highestUserWastedPercent` -> `highestUserWastedRatio`

### ISSUE THREAD

**Issue by ianfixes** [wagoodman/dive#197]
The configuration option `highestUserWastedPercent` is confusing.  It's explained in the [README.md](https://github.com/wagoodman/dive#ci-integration) as follows:

```yaml
  # Expressed as a percentage between 0-1; fails if the threshold is met or crossed.
  highestUserWastedPercent: 0.20
```

Percentages range from 0-100, not 0-1.  What you are describing here is a _ratio_.  In other words this config line does not mean `0.20%` -- it means `20%`, or a ratio of `0.2`.

I can think of 2 acceptable alternatives to this.

1. correct value as a percentage (change number)
```yaml
  # Expressed as a percentage between 0-100; fails if the threshold is met or crossed.
  highestUserWastedPercent: 20
```

2. correct value as a ratio (change name)
```yaml
  # Expressed as a ratio between 0-1; fails if the threshold is met or crossed.
  highestUserWastedRatio: 0.20
```
See also, [Verizon Math](http://verizonmath.blogspot.com/2006/12/verizon-doesnt-know-dollars-from-cents.html).

**Comment by wagoodman** [comment]
You are correct, the verbiage here is inaccurate. The upside of your proposed fixes is they address this inaccuracy, the downside is that it would unnecessarily break behavior for existing users. For now I will update the description to be a little more clear, and in the future (when bumping the major version) deprecate the current variable name and rename it to  `highestUserWastedRatio`.

**Comment by ianfixes** [comment]
> the downside [of proposed fixes is] it would unnecessarily break behavior for existing users

I agree.  I don't think this is urgent, just mildly confusing and it makes complete sense to address it if/when the next major version goes out.  Thanks for acknowledging this though!
